flujo/application/core/background_task_manager.py

```
@@ -282,15 +281,28 @@ def _context_setter(_res: Any, updated_ctx: Optional[Any]) -> None:
+                        elif "Auth" in err_name or "auth" in err_str:
+                            metadata["error_category"] = "authentication"
+                        else:
+                            metadata["error_category"] = "unknown"
                         if final_context is not None and hasattr(final_context, "scratchpad"):
-                            final_context.scratchpad["background_error_category"] = (
-                                err_ctx.category.value
-                            )
+                            final_context.scratchpad["background_error_category"] = metadata[
+                                "error_category"
+                            ]
                     except Exception:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)